pkg/imager/imager.go

```diff
@@ -251,8 +327,8 @@ func (i *Imager) buildCmdline() error {
 	cmdline.SetAll(p.KernelArgs().Strings())
 
 	// board kernel args
-	if i.prof.Board != "" {
-		var b runtime.Board
+	if i.prof.Board != "" && !quirks.New(i.prof.Version).SupportsOverlay() {
```

Comment:
I guess this would break `/promote integration-images` as `make sbcs` would no longer work? 

not a big deal for me, but we should probably remove it from the `Makefile`?

Reply:
yeh, i'll keep it commented